src/features/codeLensProvider.ts

```diff
@@ -53,23 +54,25 @@ class RunTestsCodeLens extends TestCodeLens {
     constructor(
         range: protocol.V2.Range,
         fileName: string,
+        name: string,
```

Comment:
It is the name of that particular entity, when it is a method it will be the method name, when it is the class, it will be the class name. Any suggestions ? Does `DisplayName` sound better ?

Reply:
+1, let's use a more informative name

Reply:
@rchande Got any suggestions for the name here?